storage/Devices/Bcache.h

```diff
@@ -33,80 +36,122 @@ namespace storage
     class BcacheCset;
 
 
+    /**
+     * Bcache types
+     */
+    enum class BcacheType {
+	BACKED, FLASHONLY
```

Comment:
I think FLASH_ONLY is better readable and also the standard style in C/C++.

Reply:
Done.